spec/request/rating_info_request_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'rating info', type: :request do
+  include SchemaMatchers
+
+  let(:user) { build(:disabilities_compensation_user) }
+  let(:headers) { { 'CONTENT_TYPE' => 'application/json' } }
```

Comment:
Didn't expect you to need this -- but might be in all the code -- I'll take a look.

Maybe a more descriptive name? `let(:request_json_content)....`

Reply:
Thanks, you were right - I didn't need to post so was able to remove the headers